### ISSUE TITLE
Code pointers in description

### ISSUE THREAD

**Issue by NLSurfMan** [NLSurfMan/tasks#3]

**Comment by charneykaye** [comment]
Yes! Please! This is what I'm yearning for. Is there any spec for how this would be implemented? I'm available to contribute some coding time.

**Comment by NLSurfMan** [comment]
I need to update the codebase for IDEA 2016.*. As soon as I have done that, I will see what needs to be done. Thanks for your offer to help on this. If you have a pull request, by all means, go ahead :)

**Comment by charneykaye** [comment]
Cool keep me posted!
